### PR TITLE
mostly Reviews component work

### PR DESCRIPTION
**Pull request by ghost**
mostly built out Reviews component with live data

little work on dropdown

some re-organization of /reviews route
added /reviews/meta/avg to get the avg. rating of the product (responds with a string to the tenths (eg: 4.2))

source-maps with webpack is working now
will need to "npm install"
to get the dependency package "source-map"

### CODE REVIEW

File: server/routes/products.js [hunk 1]
```diff
@@ -14,8 +14,8 @@ let ax = axios.create({
 // root ENDpoint route
 router.route('/')
   .get((req, res) => {
-    req.page = req.query.page || 1;
-    req.count = req.query.count || 5;
+    req.page = req.query.page || '1';
```
Comment (ghost) [inline server/routes/products.js:17]: template literals will coerce everything to a string
not sure what I was trying to accomplish here tbh

File: src/Reviews/Reviews.jsx [hunk 2]
```diff
@@ -1,24 +1,180 @@
...
+    let pID = `product_id=${this.state.productID}`;
+    ax.get(`/reviews/?${pID}&sort=relevant&page=1&count=2`)
+      .then((res) => {
+        this.setState({ reviews: res.data.results });
+      })
+      .catch((err) => {
+        console.dir(err);
+        console.error('err in componentdidmount.ax.get /');
+      });
+    ax.get(`/reviews/meta/?${pID}`)
```
Comment (ghost) [inline src/Reviews/Reviews.jsx:43]: this could use some major refinement

File: test_server.sh [hunk 3]
```diff
@@ -22,22 +22,27 @@ fi
 
 if [ "$1" = "reviews" ]; then
```
Comment (ghost) [inline test_server.sh:23]: I'll remove this file. Now that we have proper tests, it's no longer needed

**Comment by jameslesliecarter** [comment]
LGTM

**Pull request merged**